index.js

```diff
@@ -12,42 +9,8 @@ var AFK_SYSTEM_POLLING_TIMEOUT_MSEC = 2 * 1000;
 // percentage of the interval to be checked against the idle time. This avoid settings dynamic intervals.
 var INTERVAL_AWAY_IDLE_TIME_PERCENTAGE = 0.70;
 
-idle.tick = function (callback) {
-    callback = callback || function (){};
-
-    if (/^win/.test(process.platform)) {
-        var cmd = path.join( __dirname, 'bin', 'idle.exe');
-        execFile(cmd, function (error, stdout, stderr) {
-            if(error) {
-                callback(0, error);
-                return;
-            }
-            callback(Math.floor(parseInt(stdout, 10) / 1000), null);
-        });
-    }
-    else if (/darwin/.test(process.platform)) {
-        var cmd = '/usr/sbin/ioreg -c IOHIDSystem | /usr/bin/awk \'/HIDIdleTime/ {print int($NF/1000000000); exit}\'';
-        exec(cmd, function (error, stdout, stderr) {
-            if(error) {
-                callback(0, error);
-                return;
-            }
-            callback(parseInt(stdout, 10), null);
-        });
-    }
-    else if (/linux/.test(process.platform)) {
-        var cmd = 'xprintidle';
-        exec(cmd, function (error, stdout, stderr) {
-            if(error) {
-                callback(0, error);
-                return;
-            }
-            callback(Math.round(parseInt(stdout, 10) / 1000), null);
-        });
-    }
-    else {
-        callback(0);
-    }
+idle.tick = function () {
+    return systemIdleTime.getIdleTime() / 1000
```

Comment:
@4ver think that returns the value in seconds already.
can we remove the `tick` method?